src/components/selectbutton/SelectButton.tsx

```diff
@@ -8,12 +8,13 @@ import { setColor } from "../../utils/setcolor";
 export type SelectButtonProps = React.HTMLAttributes<HTMLButtonElement> & {
   name: string;
   value: string;
-  labelText: string;
+  labelText: string | React.ReactNode;
```

Comment:
ReactNode includes string

```suggestion
  labelText: React.ReactNode;
```